developer/js/source/lexical-model-compiler/model-definitions.ts

```diff
@@ -120,18 +120,18 @@ export class ModelDefinitions {
   /**
    * Writes out a compiled JS version of the pseudoclosure, preserving all function
    * implementations.
-   * 
+   *
    * This should be written to the file within the same IIFE as the model but BEFORE
    * the model itself, as the model will need to refer to the definitions herein.
    */
   compileDefinitions(): string {
     let defn: string = '';
-    defn += `let ${PSEUDOCLOSURE} = {\n`
+    defn += `var ${PSEUDOCLOSURE} = {\n`
 
     // ----------------------
     // START - the 'defaults', which are common within the same Developer version.
     defn += `  defaults: {\n    version: "${this.defaults.version}"`;
```

Comment:
Not that it's important for _this_ PR, but this is the site affected by #4705.

Reply:
FWIW, all of these issues were introduced in #3770. For future reference if required.